lapis/cmd/actions/annotate.moon

```diff
@@ -149,8 +149,12 @@ annotate_model = (config, fname, options={}) ->
 
   updated_source = replace_header source, header
 
-  -- TODO: this is kinda sloppy and only works with MoonScript
-  unless updated_source
+  -- TODO: this is kinda sloppy
```

Comment:
I kept the comment for no particular reason, but edited to at least not be false. 🙃

Obviously, this could be more flexible, but it does seem to work fine for now.